frontend/src/common/types/user/user-edit-form-payload.type.ts

```diff
@@ -0,0 +1,13 @@
+import { UserPayloadKey } from 'common/enums/enums';
+
+type UserEditFormPayload = {
+  [UserPayloadKey.FIRST_NAME]: string;
+  [UserPayloadKey.LAST_NAME]: string;
+  [UserPayloadKey.NICKNAME]: string;
+  [UserPayloadKey.EMAIL]: string;
+  [UserPayloadKey.BIO]: string;
+  [UserPayloadKey.BIRTHDATE]: Date;
```

Comment:
```suggestion
  [UserPayloadKey.BIRTHDATE]: string;
```

Reply:
We need a Date type for the datapicker to work.